recipe/meta.yaml

```diff
@@ -35,8 +32,9 @@ requirements:
 
 test:
   commands:
-    - test -f $PREFIX/lib/libglfw.so               # [linux]
-    - conda inspect linkages -p $PREFIX $PKG_NAME  # [linux]
+    - test -f $PREFIX/lib/libglfw.so                         # [linux]
+    - test -f $PREFIX/lib/libglfw.dylib                      # [osx]
```

Comment:
there was some kind of macro, like {{ SHEXT }} for both unixes but meh.

Reply:
So … is it okay to keep this as it is?